### DISCUSSION TITLE
No longer able to hide tabs on UI

### DISCUSSION THREAD

**Discussion by popsoda27** [vladmandic/sdnext#2122] [Q&A] [answered]
Used to be able to hide extension tabs that I don't use, like Process Image, Train, Models, Interrogator etc.
![image](https://github.com/vladmandic/automatic/assets/20932783/c061efc1-6075-4528-8d7b-a00a6950bc40)

There used to be a field in Settings>User Interface for me to specify which tabs to hide but now it's missing.
![image](https://github.com/vladmandic/automatic/assets/20932783/5e50359d-119c-4ea6-bb55-de1de44cbeef)

Is it an intended change? How do I hide tabs in this case?

**Comment by vladmandic** [comment, marked as answer]
temporarily disabled as it was causing issues with extensions compatibility.

**Reply by popsoda27** [reply]
oh ok got it! Thanks for your reply Vlad

**Reply by Faithtoken** [reply]
Is there a way to do it manually, until a fix arrives?
Maybe comment some code out or something like that.

Thank you for the great work so far by the way.
